[LPT] Documentation (developer guide) (#7444)
openvinotoolkit/openvino: docs/low_precision_transformations_ug/pipeline/step1_prerequisites.md

> # OpenVINO™ LPT: step #1. Prerequisites transformations {#openvino_docs_IE_UG_lpt_step1_prerequisites}

Reviewer: ```suggestion
# OpenVINO™ LPT: Step #1. Prerequisites Transformations {#openvino_docs_IE_UG_lpt_step1_prerequisites}
```

> Prerequisites transformations are optional. The goal is prepare a model before to run other low precision transformations. Transformations don't operate with dequantization operations, don't update precisions. Transformations:

Reviewer: ```suggestion
Prerequisites transformations are optional. The transformations prepare a model before running other low precision transformations. The transformations do not operate with dequantization operations or update precisions. Prerequisites transformations include:
```

Author: done, thanks
